src/main/io/osd.c

```diff
@@ -27,550 +29,340 @@
 
 #include "platform.h"
 
-#ifdef OSD
-
 #include "build/version.h"
 
+#ifdef OSD
+
 #include "common/utils.h"
 
 #include "drivers/system.h"
 
+#include "io/cms_types.h"
+#include "io/cms.h"
+
 #include "io/flashfs.h"
 #include "io/osd.h"
 
 #include "fc/config.h"
 #include "fc/rc_controls.h"
 #include "fc/runtime_config.h"
 
-#include "flight/pid.h"
+//#include "flight/pid.h"
 
 #include "config/config_profile.h"
 #include "config/config_master.h"
 #include "config/feature.h"
 
+// Short hands
+#define OSD_cfg masterConfig.osdProfile
```

Comment:
As long as it's code that is only for your own use, and only maintained by yourself then you are right that the code is easier to read for you.

However, for shared projects like Betaflight, where people who have never before worked on your code will have to look at your and understand it in order to debug something, when they see `masterConfig.osdProfile` they will immediately know what it is, whereas when they see `OSD_cfg` they will first have to hunt down your macro definition, making reading your code more work for everyone except you.